Library/Homebrew/formula.rb

```diff
@@ -730,12 +735,32 @@ def bottle_prefix
     prefix+".bottle"
   end
 
-  # The directory where the formula's installation logs will be written.
+  # The directory where the formula's installation or test logs will be written.
   # @private
   def logs
     HOMEBREW_LOGS+name
   end
 
+  # The prefix, if any, to use in filenames for logging current activity
```

Comment:
I don't feel strongly about this, thus feel free to leave this as-is.

This was just an idea to try and standardize a little bit on our (generally lacking) API documentation. If the description consists of complete sentences, it's fairly easy to add something to it. Otherwise I'm often wondering: Do I add a full stop now and continue with a new sentence? Do I delimit with a somewhat awkward semicolon? (And maybe I'm just overthinking such simple matters. 😀)